app/helpers/textual_mixins/textual_paused_state.rb

```diff
@@ -0,0 +1,12 @@
+module TextualMixins::TextualPausedState
+  def textual_paused_state
+    state = @record.enabled?
+    h = {:label => _("Running State"), :value => state ? _("Running") : _("Paused")}
+    if state
```

Comment:
or just `Data Collection`. I think we should use these terms in the dropdowns too.